aiopg/connection.py

```diff
@@ -421,3 +420,27 @@ def __del__(self):
     def notifies(self):
         """Return notification queue."""
         return self._notifies
+
+
+class _CursorContextManager:
```

Comment:
To make `loop.run_until_complete(conn.cursor())` works.
`iscoroutine` should pass also I believe.

Implementing `.send` etc. is required by `collections.abc.coroutine`.

`coroutines._COROUTINE_TYPES += (_RequestContextManager,)` hack is required for Python 3.4 for `run_until_complete()`, actually for `ensure_future`.
Guido approved the hack in python/asyncio#277

Requirements are relaxed by `asyncio` master a bit but released version still need that.